clear-all-data.sh

```diff
@@ -36,6 +36,11 @@ if [ -z $path_to_core ]; then
     print_usage_and_exit
 fi
 
+if docker service ls > /dev/null 2>&1; then
+  IS_LOCAL=false
+else
+  IS_LOCAL=true
+fi
```

Comment:
@euanmillar isn't this a development only script? Because we have a separate one for deployed environments in the infrastructure folder